_includes/manuals/1.7/5.2.9_cr_vmware.md

```diff
@@ -12,7 +12,7 @@ Image based provisioning can be used by provisioning a new VM from a template an
 
 #### Console access
 
-Consoles are provided using VNC connections from Foreman to the ESX server, which requires a firewall change to open the respective ports (TCP 5910 to 5930)
+Consoles are provided using VNC connections from Foreman to the ESX server, which requires a firewall change to open the respective ports (TCP 5901 to 6001)
```

Comment:
I'm unsure about this high port number - https://github.com/theforeman/foreman/blob/1.7.2/app/models/compute_resources/foreman/model/vmware.rb#L452 suggests it should only be between 5901 and 5964.